buildSrc/src/main/resources/META-INF/schemas/guide-metadata.schema.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
The JSON Schema definition does not specify whether something is required or not.

Reply:
This issue is how to handle _base_ guides, which basically only have the _publish_ property